components/policy/resources/templates/policy_definitions/README.md

```diff
@@ -0,0 +1,22 @@
+## Policy Settings in Brave
+
+These are Brave specific group policies.
+
+Our official documentation is [available here](https://support.brave.com/hc/en-us/articles/360039248271-Group-Policy).
+
+For information about adding a new policy, please see [this page](https://source.chromium.org/chromium/chromium/src/+/main:docs/enterprise/add_new_policy.md).
+
+In a nutshell, the steps for adding a new policy in Brave look like this:
+
+1. Create a new .yaml file under `//brave/components/policy/resources/templates/policy_definitions/BraveSoftware/`. The name of the file itself will be the policy name. Chromium uses capital casing. For example, we have a policy for disabling Brave Rewards called `BraveRewardsDisable.yaml`. The name used for matching is `BraveRewardsDisable`.
+2. Update the properties in that file accordingly. Look at some of the existing ones as an example.
+3. Go into `//brave/browser/policy/brave_simple_policy_map.h` and add your entry here. It'll be auto-generated as `policy::key::k` and then the policy name. With the above example, that would be `policy::key::kBraveRewardsDisable`. You must map this to a profile preference. That is what you'll check in the code.
+4. Go into `//brave/chromium_src/components/policy/resources/policy_templates.py` and add your policy name to the end of the existing string array there.
```

Comment:
I suspect this point 4. will become obsolete when you read the files dynamically.

Reply:
OK I think this should be sorted; I'll test the script and then remove this section 😄

Reply:
Removed 👍